backend/tests/services/attachment/test_encryption.py

```
@@ -0,0 +1,176 @@
+        # Act - Change the encryption key temporarily
+        with patch.dict(
+            os.environ, {"ATTACHMENT_AES_KEY": "00000000000000000000000000000000"}
+        ):
+            # Force key reload by clearing cache
+            from shared.utils import crypto
+
+            crypto._attachment_aes_key = None
+            crypto._attachment_aes_iv = None
+
+            # Assert - Should raise exception
+            with pytest.raises(Exception):
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)